docs/project/find-an-issue.md

```diff
@@ -42,68 +42,94 @@ weight=2
 
 # Find and claim an issue
 
-On this page, you choose the issue you want to work on. As a contributor, you can work
-on whatever you want. If you are new to contributing, you should start by
-working with our known issues.
+As a contributor, you can work on any open issue you want. You can view issues in the Issues tab in every repository.
+If you are new to contributing, use the filter option to find suitable issues. You can filter issues by Author, Labels, Milestones, and Assignee or sort by time such as newest and oldest. 
 
 ## Understand the issue types
+To help you identify the different types of issues, our maintainers assign labels to issues. Labels are color-coded and help you categorize and filter issues. There are four labels categories, kind, area, experience, and priority. You can filter using one or more labels. The kind and experience labels are useful for new contributors.
 
-An existing issue is something reported by a Docker user. As issues come in,
-our maintainers triage them. Triage is its own topic. For now, it is important
-for you to know that triage includes ranking issues according to difficulty. 
-
-Triaged issues have one of these labels:
+The following table describes the kind labels.
 
 <table class="tg">
-  <thead>
+<thead>
+<tr>
+<td class="tg-031e">Kind Label</td>
+<td class="tg-031e">Description</td>
+</tr>
+</thead>
+  <tbody>
     <tr>
-      <td class="tg-031e">Label</td>
-      <td class="tg-031e">Experience level guideline</td>
+      <td class="tg-031e"><strong class="gh-label kindbug">kind/bug</strong></td>
+      <td class="tg-031e">These issues contain bugs. The cause may or may not be known at triage time so debugging should be taken account into the time estimate.</td>
```

Comment:
"debugging should be taken account into..." I think that should be "taken into account.."?